pymc3/math.py

```diff
@@ -246,21 +229,37 @@ def log1mexp(x):
        "Accurately computing `\log(1-\exp(- \mid a \mid))` Assessed by the Rmpfr package"
 
     """
-    return at.switch(at.lt(x, 0.6931471805599453), at.log(-at.expm1(-x)), at.log1p(-at.exp(-x)))
+    warnings.warn(
+        "pymc3.math.log1mexp is deprecated in favor of aesara.tensor.log1mexp.\n "
```

Comment:
This is a bit confusing because there will still be a `pm.math.log1mexp` that can be used, from the user perspective, just the sign of the input changed. This is actually quite tricky, because we probably want to change the sign here too to make them compatible. As this is a new major version, I think the best way would be to raise a warning and say that they need to invert the input as the interface changed.

Reply:
Or we also add a `negative_input` flag here and deprecate that way.

Reply:
I was thinking that we would deprecate it for one version and then bring it back, but yes, maybe that's more sensible.

Reply:
Updated to follow the same logic as the `numpy` version. Later on we should deprecate the keyword argument and finally remove the function...